go.mod

```diff
@@ -0,0 +1,75 @@
+module github.com/ONSdigital/dp-cantabular-metadata-extractor-api
+
+go 1.18
+
+// Fix security issues
+replace github.com/spf13/cobra => github.com/spf13/cobra v1.4.0
+
+require (
+	// point dp-api-clients-go at feature/2021-md-stopgap branch which we don't want to merge yet
+	github.com/ONSdigital/dp-api-clients-go/v2 v2.142.1-0.20220715151843-809f8aa64763
```

Comment:
Should we use a proper version now?

Reply:
See https://github.com/ONSdigital/dp-api-clients-go/pull/342  (and further change needed in this repo)